### PR TITLE
ISSUE #3247 Fix minishift openshift component list Description

### LINKED ISSUE #3247: In correct documentation for component list
The description for ` minishift openshift component add` and ` minishift openshift component list ` are the same. It looks like a copy paste error.  
image

### PR DESCRIPTION
**Pull request by danielhelfand**
Fixes #3247 

This pull request updates the description for `minishift openshift component list`. It is currently the same as `minishift openshift component add`. I based the description off of what is in this [documentation](https://github.com/minishift/minishift/blob/master/docs/source/openshift/openshift-client-binary.adoc).

Steps to test the pull request:

  1. Verify using `minishift openshift component --help` that the description for `list` is as follows:
```
Available Commands:
  add      Add component to an OpenShift cluster (Works only with OpenShift version >= 3.10.x)
  list        List valid components that can be added to an OpenShift cluster (Works only with OpenShift version >= 3.10.x)
```

### CODE REVIEW

**Comment by danielhelfand** [comment]
ok to test

File: cmd/minishift/cmd/openshift/component_list.go [hunk 1]
```diff
@@ -27,8 +27,8 @@ import (
 // version command represent current running openshift version and available one.
 var componentListCmd = &cobra.Command{
 	Use:   "list [component-name]",
-	Short: "Add component to an OpenShift cluster (Works only with OpenShift version >= 3.10.x)",
-	Long:  "Add component to an OpenShift cluster (Works only with OpenShift version >= 3.10.x)",
+	Short: "List valid components that can be added to an OpenShift cluster (Works only with OpenShift version >= 3.10.x)",
```
Comment (praveenkumar) [inline cmd/minishift/cmd/openshift/component_list.go]: 👍

**Comment by danielhelfand** [comment]
Recent push was just changing commit message to include issue number.

**Comment by praveenkumar** [comment]
ok to test

**Comment by anjannath** [comment]
merged in https://github.com/minishift/minishift/commit/22359480246cec3a116c6d9d7201c6d48bba0f43

**Pull request closed without merging**